### DISCUSSION TITLE
The interface has stopped working

### DISCUSSION THREAD

**Discussion by C-ajal** [Security-Onion-Solutions/securityonion#10695] [General]
I installed distributed, and upon completion of the installation and reboot, the web interface can be accessed for 5 minutes, then a message appears (cannot attmpte with manager), after which the system falls, and when the address is requested, a message appears to me (This site can’t be reached)
Note that when executing the command (so-status), all services are ok

**Comment by Acewiza** [comment]
Try waiting longer than 5 min.  That interface has [more layers of onion](https://www.google.com/search?client=firefox-b-1-e&sxsrf=AB5stBgNZ0EXACYVyM7VqxsBq0CxyAQqNA:1688563841425&q=image+high+stacked+onion+burger&tbm=isch&source=univ&fir=1QQbmRxRyf0aMM%252CHbLHS8djD9QTTM%252C_%253Bt6BHK9uYN4sdmM%252CkMMipb7dARYMrM%252C_%253B0gawt31smFGGjM%252CgXXFzZYuW-iIfM%252C_%253BRaoun70xphdxEM%252Coao3PRbD129WOM%252C_%253Bg0gpuRvctlsGdM%252CTPdf9A50SMlUvM%252C_%253BJr6gNpVf2WvLvM%252C2FZiz8qFCEF5XM%252C_%253Bw7P_7m4IUakC_M%252Cgr1SAYsiMMZqWM%252C_%253BtwqPCzUr_5_ztM%252Ct5vYmea7t5U_LM%252C_%253BOuIH7G9C4deymM%252Crb3WukuNSZd4HM%252C_%253Bwgs9j2UeTaPWOM%252CDc18UaCf52c3ZM%252C_&usg=AI4_-kRTsVoLEMPX9oc7x4KkHJ9ir9DS5A&sa=X&ved=2ahUKEwj0lsu_1vf_AhV0IzQIHRFNA1IQjJkEegQIOBAC&biw=2560&bih=1292&dpr=1#imgrc=Jr6gNpVf2WvLvM) stacked on it than you could possibly imagine.
